Dockerfile

```diff
@@ -0,0 +1,14 @@
+ARG PYTHON_VERSION
+
+FROM python:${PYTHON_VERSION}
+
+RUN apt-get update -y && apt-get install -y apt-utils tox
```

Comment:
Would it be OK to use the Python `-alpine` images here for smaller image download size?

Install command would be `apk update && apk add --no-cache py3-tox` in this case.